ocaml-lsp-server/test/e2e-new/document_flow.ml

```diff
@@ -0,0 +1,55 @@
+open Test.Import
+
+let%expect_test "double open a document" =
```

Comment:
I'd change the name of the text to something that's actually describing an expected behavior.
E.g. "[...] should allow double opening the same document"

Current description requires reading all test implementation to figure out expected outcome